Applications/WebPortal/WebApp/library/source/pages/account_page/risk_page/risk_parameters_view.tsx

```diff
@@ -122,18 +124,35 @@ export class RiskParametersView extends React.Component<Properties> {
         <Padding/>
       </HBoxLayout>);
   }
+  
+  private copyParameter() {
+    const oldParameters = this.props.parameters;
+    const parameters = new Nexus.RiskParameters(
```

Comment:
```RiskParameters``` already has a ```clone``` method that will do this for you.

Reply:
How do I use it? I tried `this.props.parameters.copy() `and `this.props.parameters.clone()` and either seem to work. And Nexus.RiskParameters doesn't have a clone method.

Reply:
Okay thanks for pointing this out. Merge master into your local branch and use the ```clone``` method.

Reply:
In the future if you come across a situation where you are making a copy like this, let me know as it indicates we should be adding a ```clone``` method to the class.